draft-ietf-quic-transport.md

```diff
@@ -1992,7 +1992,7 @@ Error Code:
 Receivers send ACK frames to inform senders which packets they have received and
 processed, as well as which packets are considered missing.  The ACK frame
 contains between 1 and 256 ACK blocks.  ACK blocks are ranges of acknowledged
-packets. Implementations SHOULD NOT generate ACK packets in response to packets
+packets. Implementations MUST NOT generate ACK packets in response to packets
```

Comment:
Hmm, this is a fine change in principle, but the sentence is nonsensical.  "ACK packets"?

Isn't the change we want something like: "MUST NOT generate a packet containing only ACK frames if all they have received is packets containing ACK frames."

Reply:
Yes, of course.  That packet text is everywhere.